notebooks/nbconverted/VAE_test.py

```diff
@@ -4,20 +4,24 @@
 # # Image-based profiles VAEs
 
 # In[1]:
-
+import sys
+import warnings
 
 import numpy as np
 import pandas as pd
 import torch
 import torch.nn as nn
 import torch.optim as optim
-from cytotraj.utils.model_utils import load_data
 from torch.utils.data import Dataset
 
+warnings.simplefilter("ignore")
+sys.path.append("../")
+from cytotraj.utils.io import load_data  # noqa
```

Comment:
Does this fix #4?

Reply:
yes! the module is created under cytotraj/utils/io.py